docs/assignment1.md

```diff
@@ -1,62 +1,75 @@
+| F5   | Map        |    A list that keeps track of the places already visited     |
+| F6   | Backpack   | A list that keeps track of the items that have been picked up already |
+| F7   | Score      | An integer that shows the number of already obtained points; It will be updated after every action |
+| F8   | Moves      | An integer that shows the number of moves. Always updated after an action |
+| F9   | Timer      | A clock that shows the time that has already been passed since the start of the game |
+| F10  | Rooms      | There is a specific number of rooms that the player can enter, each resembling a room at the VU. |
+|      |            |                                                              |
 
 ### Quality requirements
-Author(s): `name of the team member(s) responsible for this section`
+
+Author(s): Irene Garcia-Fortea Garcia, Florent Brunet de Rochebrune 
 
 As a preamble to the table, you can discuss the main line of reasoning you followed for coming up with the quality requirements listed below.
 
 | ID  | Short name  | Quality attribute | Description  |
 |---|---|---|---|
-| QR1  | Commands sanity checks | Reliability  | When the player issues a command, the syntax of the command shall always get validated against the format specified in F2 |
-| QR2  | Extensible world | Maintainability  | The video game shall be easilty extendable in terms of levels, worlds, interaction points  |
-| QR3  | Instantaneous results | Responsiveness  | Once the scores of all soccer players are provided by the user, the results of the virtual matches shall be available within 1 second |
-| QR4  | ... | ... | ... |
-
-Each quality requirement must be tagged with the corresponding quality attribute (see corresponding slides of the first lecture for knowing them).
-
-Maximum number of words for this section: 1000
+| QR1  | Check for valid moves and valid input | Correctness | Check if the move the user wants to do is possible at that time of the game and check whether the input was given in the correct syntax. If the command does not meet the requirement: an error message must be printed and/or the user must be asked to give a new input. |
+| QR2  | Quick     responses to user input     | Responsiveness | The system should give a fast response (and always give a response) to the user so that the game flow will not be interrupted. If the user obtained a point, the score should be updated within 1 second. |
+| QR3  | Extensible world | Maintainability | The code should be well structured and organized; and the universe should not be closed. It should be an open story plot so more levels can be added later. Readable code will make it easier to implement such. |
+| QR4  | User friendly | Usability | An introduction must explain how to play the game and how to give commands. A well organized structure will make it easier for the player to keep an overview of the game. |
```

Comment:
user friendly is a generic concept. precisely define the features you want to implement